src/main/drivers/accgyro/accgyro_l3gd20.c

```diff
@@ -15,6 +15,8 @@
  * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#ifdef USE_GYRO_L3GD20
```

Comment:
This has to be after `platform.h`, or else it will be before the include for `target.h`.